warehouse/locale/messages.pot

```diff
@@ -3009,125 +3252,140 @@ msgstr ""
 msgid "Back to projects"
 msgstr ""
 
+#: warehouse/templates/manage/organizations.html:22
+#: warehouse/templates/manage/projects.html:22
+#, python-format
+msgid "Pending invitations (%(invitation_count)s)"
```

Comment:
I think this will read better as

"(%(invitation_count)s) pending invitations"

or as

"Pending invitations: (%(invitation_count)s)" [note the colon]